pahfit/base.py

```diff
@@ -363,27 +399,55 @@ def plot(self, axs, x, y, yerr, model, model_samples=1000, scalefac_resid=2):
         # now plot the dust bands and lines
         for cmodel in model:
             if isinstance(cmodel, Gaussian1D):
-                ax.plot(x_mod, (cont_y + cmodel(x_mod)) * ext_model / x_mod, "#DC267F", alpha=0.5)
+                ax.plot(
+                    x_mod,
+                    (cont_y + cmodel(x_mod)) * ext_model / x_mod,
+                    "#DC267F",
+                    alpha=0.5,
+                )
             if isinstance(cmodel, Drude1D):
-                ax.plot(x_mod, (cont_y + cmodel(x_mod)) * ext_model / x_mod, "#648FFF", alpha=0.5)
+                ax.plot(
+                    x_mod,
+                    (cont_y + cmodel(x_mod)) * ext_model / x_mod,
+                    "#648FFF",
+                    alpha=0.5,
+                )
 
         ax.plot(x_mod, cont_y * ext_model / x_mod, "#785EF0", alpha=1)
 
         ax.plot(x_mod, model(x_mod) / x_mod, "#FE6100", alpha=1)
-        ax.errorbar(x, y / x, yerr=yerr / x,
-                    fmt='o', markeredgecolor='k', markerfacecolor='none',
-                    ecolor='k', elinewidth=0.2, capsize=0.5, markersize=6)
+        ax.errorbar(
+            x,
+            y / x,
+            yerr=yerr / x,
+            fmt="o",
+            markeredgecolor="k",
+            markerfacecolor="none",
+            ecolor="k",
+            elinewidth=0.2,
+            capsize=0.5,
+            markersize=6,
+        )
 
         ax.set_ylim(0)
         ax.set_ylabel(r"$\nu F_{\nu}$")
 
-        ax.legend(Leg_lines, ["S07_attenuation",
-                              "Spectrum Fit",
-                              "Dust Features",
-                              r"Atomic and $H_2$ Lines",
-                              "Total Continuum Emissions",
-                              "Continuum Components"], prop={'size': 10}, loc="best", facecolor="white", framealpha=1,
-                  ncol=3)
+        ax.legend(
+            Leg_lines,
+            [
+                "S07_attenuation",
+                "Spectrum Fit",
+                "Dust Features",
+                r"Atomic and $H_2$ Lines",
```

Comment:
Typo?   r"Atomic and $H_2$ Lines"

Reply:
The "r" is in front to allow indicate that there is latex ahead in the string.  See https://www.pythontutorial.net/python-basics/python-raw-strings/.

Reply:
Ah. Thanks!